backend/__tests__/userGetBookings.test.js

```
@@ -0,0 +1,179 @@
+const mongoose = require('mongoose');
+const jwt = require('jsonwebtoken');
+const httpMocks = require('node-mocks-http');
+const { getBookings } = require('../controllers/userControllers'); // Adjust path to match your actual file
+const { Booking } = require('../models/bookingSchemas');
+
+// Mock the models and dependencies
+jest.mock('../models/bookingSchemas');
+jest.mock('jsonwebtoken');
+
+describe('getBookings Controller', () => {
+  let req, res;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)